router/core/graphql_prehandler.go

```diff
@@ -200,6 +200,11 @@ func (h *PreHandler) Handler(next http.Handler) http.Handler {
 		// AFTER we're done with body (retrieved from parser.ReadBody())
 		buf := h.getBodyReadBuffer(r.ContentLength)
 
+		_, operationReadSpan := h.tracer.Start(r.Context(), "Operation - Read",
```

Comment:
I'd trace a regular query from a file upload request. This makes investigations easier.